automl/automl.q

```diff
@@ -6,7 +6,7 @@
 
 
 \d .automl
-\l shim/shim.q
+\l ml/shim.q
```

Comment:
Would it be worth changing this to be a load after the `loadfile` definition and changing to something like
```q
.automl.loadfile`$"../ml/shim.q"
```
(I think that works) basically so that the loading of the library can be anywhere that the repo is stored, doing it using the current approach requires that either the user is in the directory just below this with `ml` folder also available, changing it to this will make it so users should be able to load from any relative path.